felix/bpf/cleanup.go

```diff
@@ -28,7 +28,8 @@ func CleanUpCalicoPins(dir string) {
 		if err != nil {
 			return err
 		}
-		if strings.HasPrefix(info.Name(), "cali_") || strings.HasPrefix(info.Name(), "calico_") {
+		if strings.HasPrefix(info.Name(), "cali_") || strings.HasPrefix(info.Name(), "calico_") ||
```

Comment:
Worth a utility function to share the match between these two files?

Reply:
Ack, but the parsing is done differently in the test. and atm it is imo just fine.